yarn.lock

```diff
@@ -8599,6 +8599,15 @@ axe-core@^4.2.0:
   resolved "https://registry.yarnpkg.com/axe-core/-/axe-core-4.3.5.tgz#78d6911ba317a8262bfee292aeafcc1e04b49cc5"
   integrity sha512-WKTW1+xAzhMS5dJsxWkliixlO/PqC4VhmO9T4juNYcaTg9jzWiJsou6m5pxWYGfigWbwzJWeFY6z47a+4neRXA==
 
+axios@1.6.3:
```

Comment:
Looks like v1.6.0 is still listed below as well. Might just need a clean bootstrap?

Reply:
ahh good catch. Ill add it back

Reply:
added in latest commit

Reply:
Cleaned in latest commit